### PR TITLE
Fix per Zen's suggestion: disable aggregation in default

### PR DESCRIPTION
**Pull request by feng-tao**
In current Naarad, there is a bug in the code that even if user wants to aggregate metric in "minute" or "hour", the code wont pick up this option. And it aggregate value in "second" per default.  Per Zen's suggestion, if we have sub-second value, the code will only print out the average value.

This pull request disable this per-second aggregation unless user specify "aggr_metrics=second/minute/hour" in metric section. I try the fix which works on the config provided by Zen.

### CODE REVIEW

**Comment by richardhsu** [comment]
As per standup discussion, it would be best to not disable by default as some previous configurations may not expect this and may break those usages of Naarad. Instead flip it instead to have an option to specify that aggregation should be None to just take whatever is given.

File: src/naarad/metrics/gc_metric.py [hunk 1]
```diff
@@ -52,9 +52,9 @@ class GCMetric(Metric):
       'g1-survivor-after-gc', 'g1-heap-occupancy-before-gc', 'g1-heap-capacity-before-gc', 'g1-heap-occupancy-after-gc', 'g1-heap-capacity-after-gc',
       'g1-young-cpu.sys', 'g1-young-cpu.usr', 'g1-young-cpu.real', 'g1-mixed-cpu.usr', 'g1-mixed-cpu.sys', 'g1-mixed-cpu.real')
 
-  def __init__ (self, metric_type, infile_list, hostname, outdir, resource_path, label, ts_start, ts_end, rule_strings,
+  def __init__ (self, metric_type, infile_list, hostname,aggr_metrics, outdir, resource_path, label, ts_start, ts_end, rule_strings,
```
Comment (RiteshMaheshwari) [inline src/naarad/metrics/gc_metric.py]: space after comma please

**Comment by feng-tao** [comment]
Thanks for all the suggestions. Just update the pull request.

**Comment by feng-tao** [comment]
One thing to comment: to disable the aggregation, user needs to specify "aggr_metrics=none" in the configure file. Do you have any suggestions of a nicer way?

**Comment by RiteshMaheshwari** [comment]
Sounds good. Just make sure that 'none' 'None', 'NONE' all are easily taken in.

**Comment by feng-tao** [comment]
Is this pull request ready to be check-in?

**Comment by richardhsu** [comment]
Sorry Tao. I think I'll dedicate some time tomorrow to work on Naarad and work through the pull requests and get a cleaner version of Naarad pushed to Pypi so this will be one of the ones I review.

**Comment by feng-tao** [comment]
Thanks Richard~

**Comment by RiteshMaheshwari** [comment]
Cannot be automatically merged, right now. Can you please fix? After that, @richardhsu would you have time to look at this?

**Comment by richardhsu** [comment]
I'll handle this right now as I think the conflicts are with an update I made.

**Comment by feng-tao** [comment]
Thanks Richard.
-Tao

**Pull request merged**